packages/web/components/Landing/sections/Cards.tsx

```diff
@@ -0,0 +1,138 @@
+import { Box, HStack, Image, ListItem, OrderedList, Text } from '@metafam/ds';
+import CardBackground from 'assets/landing/card-background.png';
+import CardImage from 'assets/landing/card-image.png';
+
+function Intro() {
+  return (
+    <Box
+      width="100%"
+      minHeight="1040px"
+      background="#1B0D2A"
```

Comment:
These colors should be using chakra theme colors instead of hex codes. If the color is not already there, consider adding it to design-system